src/ORM.php

```diff
@@ -179,15 +179,10 @@ public function make(string $role, array $data = [], int $node = Node::NEW)
         );
     }
 
-    /**
-     * @inheritdoc
-     */
+    /** @deprecated since Cycle ORM v1.8, this method will be removed in future releases. Use method with instead. */
```

Comment:
```suggestion
    /** @deprecated since Cycle ORM v1.8, this method will be removed in future releases. Use method {@see with} instead. */
```

This can be more helpful